vizro-core/src/vizro/managers/_data_manager.py

```diff
@@ -21,12 +25,15 @@ class DataManager:
 
     """
 
+    _cache = Cache(config={"CACHE_TYPE": "NullCache"})
```

Comment:
Note to myself to think about whether we want `NullCache` or `SimpleCache` as default and whether we want some sort of development vs. production mode that switches cache automatically or gives a hint...